.eslintrc-js.json

```diff
@@ -27,7 +27,8 @@
         "import/no-extraneous-dependencies": [
           "error",
           {
-            "devDependencies": ["src/__tests__/**/*.js"]
+            "devDependencies": ["src/__tests__/**/*.js"],
+            "packageDir": ["./", "../../"]
```

Comment:
This will read dev dependencies from the root `package.json` as well, and will not mark them as missing in tests.

![image](https://github.com/beyondessential/tupaia/assets/20692464/22f74557-c49a-4bbc-8c16-3ac4c7ffddaa)